### PR TITLE
feat: BioNemo EKS cluster Using Cloudwatch Observability Plugin

### PR DESCRIPTION
**Pull request by JohnDzialo**
### What does this PR do?

🛑 Please open an issue first to discuss any significant work and flesh out details/direction - we would hate for your time to be wasted.
Consult the [CONTRIBUTING](https://github.com/awslabs/data-on-eks/blob/main/CONTRIBUTING.md#contributing-via-pull-requests) guide for submitting pull-requests.

* update ai-ml bionemo project to include some training exercises on ESM2 model.
* add k8s metric server deployment to enable kubectl top commands
* turn off cloudwatch agents deployment
* turn on cloudwatch observability add-on to include NVIDIA GPU and Neuron metrics
* add policy to node groups, CloudWatchAgentServerPolicy, to allow cloudwatch agents to access Cloudwatch from cluster
* update GPU ML AMI to latest to have the latest NVIDIA drivers and nvidia-container-toolkit
* README to describe how to install kubeflow operator to run bionemo PyTorch jobs as a k8s CRD
* update instance type to g5.12xlarge, p3s are hard to get and take away from customers

### Motivation

Run BioNemo Training and observe the Container Insights including GPU/Neuron metrics

### More

- [x] Yes, I have tested the PR using my local account setup (Provide any test evidence report under Additional Notes)
- [x] Mandatory for new blueprints. Yes, I have added a example to support my blueprint PR
- [ ] Mandatory for new blueprints. Yes, I have updated the `website/docs` or `website/blog` section for this feature
- [x] Yes, I ran `pre-commit run -a` with this PR. Link for installing [pre-commit](https://pre-commit.com/) locally

### For Moderators

- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

**Review by askulkarni2 — COMMENTED** [review]
Assuming this is a successor to esm-1nv? Please update the website https://awslabs.github.io/data-on-eks/docs/gen-ai/training/GPUs/bionemo accordingly. Not sure if we need to deprecate esm-1nv.

cc: @dorukozturk

**Comment by JohnDzialo** [comment]
Thanks @askulkarni2 

I have added the requested changes. Let me know what you think.

**Comment by JohnDzialo** [comment]
Where can I update the docs?

File: ai-ml/bionemo/eks.tf [hunk 1]
```diff
@@ -109,12 +109,12 @@ module "eks" {
       )
 
       ami_type            = "AL2_x86_64_GPU"
-      ami_release_version = "1.29.0-20240213"
+      ami_release_version = "1.29.6-20240828"
```
Comment (vara-bonthu) [inline ai-ml/bionemo/eks.tf:112]: We don't have to specify the `ami_release_version` verison. Module will always get the latest AMI for the ami_type

**Review by vara-bonthu — COMMENTED** [review]
One minor comment otherwise lgtm

File: ai-ml/bionemo/eks.tf (hunk 1 above)
Comment (JohnDzialo) [reply ai-ml/bionemo/eks.tf]: @vara-bonthu updated to remove the ami_release_version.

I also fixed the pre-commit failures.

**Comment by JohnDzialo** [comment]
@vara-bonthu @askulkarni2 

Anything else I need for this PR before merging?

**Review by vara-bonthu — APPROVED** [review]

**Pull request merged**